packages/trader/src/App/Containers/trade-footer-extensions.jsx

```diff
@@ -14,6 +14,7 @@ const TradeFooterExtensions = ({
     togglePositionsDrawer,
 }) => {
     React.useEffect(() => populateFooter());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
can put "off" in .eslintrc for this lint type

Reply:
Usually disabling it is a mistake and will significantly affect the performance so it is good to actually get this errors and fix it in some cases.